<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/data.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="list">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Playback</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Behavior</string>
      </property>
      <property name="icon">
       <iconset resource="../../data/data.qrc">
        <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Song Information</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Global Shortcuts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notifications</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Music Library</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last.fm</string>
      </property>
      <property name="icon">
       <iconset resource="../../data/data.qrc">
        <normaloff>:/last.fm/as.png</normaloff>:/last.fm/as.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magnatune</string>
      </property>
      <property name="icon">
       <iconset resource="../../data/data.qrc">
        <normaloff>:/magnatune.png</normaloff>:/magnatune.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="title"/>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stacked_widget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="playback_page">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QCheckBox" name="current_glow">
           <property name="text">
            <string>Show a glowing animation on the current track</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Fading</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="fading_out">
              <property name="text">
               <string>Fade out when stopping a track</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fading_cross">
              <property name="text">
               <string>Cross-fade when changing tracks manually</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="fading_auto">
              <property name="text">
               <string>Cross-fade when changing tracks automatically</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="fading_options" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Fading duration</string>
                 </property>
                 <property name="indent">
                  <number>22</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="fading_duration">
                 <property name="suffix">
                  <string> ms</string>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="singleStep">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>257</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="replaygain_group">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Replay Gain</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="replaygain">
              <property name="text">
               <string>Use Replay Gain metadata if it is available</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Replay Gain mode</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="replaygain_mode">
                 <item>
                  <property name="text">
                   <string>Radio (equal loudness for all tracks)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Album (ideal loudness for all tracks)</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Pre-amp</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="replaygain_preamp_label"/>
                 </item>
                 <item>
                  <widget class="StickySlider" name="replaygain_preamp">
                   <property name="maximum">
                    <number>300</number>
                   </property>
                   <property name="value">
                    <number>150</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sticky_center" stdset="0">
                    <number>150</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="replaygain_compression">
                 <property name="text">
                  <string>Apply compression to prevent clipping</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gst_group">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>GStreamer audio engine</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Output plugin</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="gst_plugin">
              <item>
               <property name="text">
                <string>Choose automatically</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="LineEdit" name="gst_device">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="hint" stdset="0">
               <string>Leave blank for the default.  Examples: &quot;/dev/dsp&quot;, &quot;front&quot;, etc.</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="gst_device_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Output device</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QCheckBox" name="b_show_tray_icon_">
           <property name="text">
            <string>Show tray icon</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>When Clementine starts</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="b_always_show_">
              <property name="text">
               <string>Always show the main window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="b_always_hide_">
              <property name="text">
               <string>Always hide the main window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="b_remember_">
              <property name="text">
               <string>Remember from last time</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Language</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QComboBox" name="language">
              <item>
               <property name="text">
                <string>Use the system default</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>You will need to restart Clementine if you change the language.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>290</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="song_info_page">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Lyrics</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="LyricSettings" name="lyric_settings" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="GlobalShortcutsConfig" name="global_shortcuts" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="notifications_page">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Clementine can show a message when the track changes.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Notification type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="notifications_none">
              <property name="text">
               <string>Disabled</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="notifications_native">
              <property name="text">
               <string>Show a native desktop notification</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="notifications_pretty">
              <property name="text">
               <string>Show a pretty OSD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="notifications_tray">
              <property name="text">
               <string>Show a popup from the system tray</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="notifications_general">
           <property name="title">
            <string>General settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QWidget" name="notifications_options" native="true">
              <layout class="QFormLayout" name="formLayout">
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Popup duration</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="notifications_duration">
                 <property name="suffix">
                  <string> seconds</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notifications_volume">
              <property name="text">
               <string>Show a notification when I change the volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notifications_art">
              <property name="text">
               <string>Include album art in the notification</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="notifications_pretty_group">
           <property name="title">
            <string>Pretty OSD options</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Background opacity</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="notifications_opacity">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Background color</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="notifications_bg_preset">
              <item>
               <property name="text">
                <string>Basic Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Clementine Orange</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom...</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Text color</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="notifications_fg_choose">
              <property name="text">
               <string>Choose color...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>286</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="library_page">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="LibraryConfig" name="library_config" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="lastfm_page">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="LastFMConfig" name="lastfm" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="magnatune_page">
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="MagnatuneConfig" name="magnatune" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>LastFMConfig</class>
   <extends>QWidget</extends>
   <header>radio/lastfmconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryConfig</class>
   <extends>QWidget</extends>
   <header>library/libraryconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StickySlider</class>
   <extends>QSlider</extends>
   <header>widgets/stickyslider.h</header>
  </customwidget>
  <customwidget>
   <class>MagnatuneConfig</class>
   <extends>QWidget</extends>
   <header>radio/magnatuneconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GlobalShortcutsConfig</class>
   <extends>QWidget</extends>
   <header>ui/globalshortcutsconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LyricSettings</class>
   <extends>QWidget</extends>
   <header>songinfo/lyricsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>fading_out</tabstop>
  <tabstop>fading_cross</tabstop>
  <tabstop>fading_auto</tabstop>
  <tabstop>fading_duration</tabstop>
  <tabstop>replaygain</tabstop>
  <tabstop>replaygain_mode</tabstop>
  <tabstop>replaygain_preamp</tabstop>
  <tabstop>replaygain_compression</tabstop>
  <tabstop>gst_plugin</tabstop>
  <tabstop>gst_device</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>notifications_pretty</tabstop>
  <tabstop>notifications_tray</tabstop>
  <tabstop>notifications_duration</tabstop>
  <tabstop>notifications_volume</tabstop>
  <tabstop>notifications_art</tabstop>
  <tabstop>notifications_opacity</tabstop>
  <tabstop>notifications_bg_preset</tabstop>
  <tabstop>notifications_fg_choose</tabstop>
  <tabstop>notifications_native</tabstop>
  <tabstop>b_remember_</tabstop>
  <tabstop>notifications_none</tabstop>
  <tabstop>b_show_tray_icon_</tabstop>
  <tabstop>b_always_hide_</tabstop>
  <tabstop>b_always_show_</tabstop>
 </tabstops>
 <resources>
  <include location="../../data/data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stacked_widget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>549</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaygain</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
